roadmap.qmd

```diff
@@ -4,155 +4,213 @@ title: Roadmap
-| {{< var status.potential >}} | TBD | Theme customization extensions | Plugin/extension system for theming of user interface elements of all Seedcase software |
-
-: Secondary software deliverables with their approximate timelines for
-completion of a minimum viable product (prototype) for each.
-
-### Documentation
-
-| Status | Timeline | Deliverable | Description |
-|------------------|------------------|------------------|--------------------|
-| {{< var status.wip >}} | Ongoing | Knowledge sharing via the [Community](https://community.seedcase-project.org/) website | Knowledge and skills learned from attending conferences and workshops in research software and data engineering or operational management |
-| {{< var status.planned >}} | TBD | Opinion paper | Workflows and digital infrastructure for building software in a team-based research environment |
-
-: Secondary documentation deliverables with their approximate timelines for
-uploading to an archive for the first time.
+These are activities that would involve engaging with the research
+community, mainly through in-person seminars, symposiums, or other
+events.
 
-### Training and education
-
-| Status | Timeline | Deliverable | Description |
-|------------------|------------------|------------------|--------------------|
-| {{< var status.wip >}} | Mid-2025 | [`rostools/r-pkg-intro`](https://github.com/rostools/r-pkg-intro) | An introduction course to building R Packages |
-| {{< var status.planned >}} | End 2025 | [`rostools/cog-flow-intro`](https://github.com/rostools/cog-flow-intro) | :gear: Collaboration with Git and GitHub: A gentle introduction to a team-based collaborative workflow using Git and GitHub. |
-| {{< var status.potential >}} | TBD | Course on project development | Introductory course on iterative project management and development for effective collaboration and work. |
-| {{< var status.potential >}} | TBD | Seminar series | What is research data engineering? |
-| {{< var status.potential >}} | TBD | Workshop | Data engineering in a research environment |
-
-: List of secondary deliverables with approximate timelines for running
-the first workshop or course.
+::: callout-note
+Given the expertise and resources needed for these types of "products"
+(e.g. hosting and coordinating people), we will likely not be able to
+work on these types of activities in the near future. However, we
+believe that they are important to bring greater awareness to the
+research community about the various topics we work on and advocate for.
 :::
 
-## Ongoing activities
+| Status | Deliverable |
+|-----------------------------------------------|------------------------|
+| {{< var status.potential >}} | Seminar series: Unmet needs for research data engineering and current challenges in implementation |
+| {{< var status.potential >}} | Symposium: Current state of data engineering in research environment |
```

Comment:
I don't think we can actually do these, but  I feel like it would give us more insight into what people on the ground are struggling with and to start building a community around it.